modules/ipccar5/icesheets/ipccar5_icesheets_project.py

```diff
@@ -237,26 +241,27 @@ def ar5_project_icesheets(rng_seed, pyear_start, pyear_end, pyear_step, cyear_st
 	nyr = len(data_years)
 
 	# correlation between antsmb and antdyn
-	fraction=np.random.rand(nmsamps * ntsamps)
+	#fraction=np.random.rand(nmsamps * ntsamps)
+	fraction = np.random.rand(nsamps)
 
 	# Project the SMB and Dynamics portions of each ice sheet
-	greensmb=project_greensmb(zt, my_fit, nmsamps)
-	greendyn=project_greendyn(my_fit, nmsamps, ntsamps, data_years)
-	antsmb=project_antsmb(zit, my_fit, nmsamps, ntsamps, fraction=fraction)
-	antdyn=project_antdyn(my_fit, nmsamps, ntsamps, data_years, fraction=fraction)
+	greensmb=project_greensmb(temp_samples, my_fit, nsamps)
+	greendyn=project_greendyn(my_fit, 1, nsamps, data_years)
+	antsmb=project_antsmb(inttemp_samples, my_fit, 1, nsamps, fraction=fraction)
+	antdyn=project_antdyn(my_fit, 1, nsamps, data_years, fraction=fraction)
 
 	# Center the projections to the baseyear
 	baseyear_idx = np.flatnonzero(data_years == startyr)
 	if len(baseyear_idx) > 0:
-		greensmb = greensmb - greensmb[:,:,baseyear_idx]
+		greensmb = greensmb - greensmb[:,baseyear_idx]
 		greendyn = greendyn - greendyn[:,:,baseyear_idx]
 		antsmb = antsmb - antsmb[:,:,baseyear_idx]
 		antdyn = antdyn - antdyn[:,:,baseyear_idx]
 
 	# Reduce the years to just the target years
 	year_idx = np.isin(data_years, targyears)
 	data_years = data_years[year_idx]
-	greensmb = greensmb[:,:,year_idx]
+	greensmb = greensmb[:,year_idx]
```

Comment:
Output seems ok but it's weird that dimensionality is different among terms here - what is going on?